srcpkgs/power-profiles-daemon/files/power-profiles-daemon/run

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
 exec 2>&1
+
+# In order to prevent unpredictable behaviour,
+# we check whether 'tlp' is running
+sv check tlp >/dev/null && exit 1
```

Comment:
This should probably be `sv status` instead of `sv check` since the latter only returns true if the current service state matches the requested state. (meaning that it would return false for `wants: up, is: down` and `wants: down, is: up`)